src/pages/Home.js

```diff
@@ -1,9 +1,13 @@
 
 import styled from "styled-components";
 import { useNavigate } from "react-router-dom";
+import { useState } from "react";
 
 export default function Home() {
   const navigate = useNavigate();
+  const [buttonBox1, setButtonBox1] = useState(false);
+  const [buttonBox2, setButtonBox2] = useState(false);
+  const [buttonBox3, setButtonBox3] = useState(false);
   const user = localStorage.getItem(`user`);
 
   if(user === null) {
```

Comment:
This is a side effect, which should be handled in a useEffect

Reply:
in case somebody enter over the link /home without going over /login. So if he is not logged in h should not be able to use anything.

There might be a better solution to do this.